.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
❗️ This matches some files in source control, but not others. I would prefer to limit this to the ones we know, which would probably start with this:

```
[*.{java,stg}]
```